.github/workflows/regression.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: CMake configure
         run: |
           mkdir build
-          CC=clang cmake . -DCMAKE_BUILD_TYPE=Debug -DWITH_UNIT_TESTS=ON -B build
+          CC=clang cmake . -DCMAKE_LINKER=ld -DCMAKE_BUILD_TYPE=Debug -DWITH_UNIT_TESTS=ON -B build
```

Comment:
Huh. It's `CC` for compiler, but `LD` for linker?

Reply:
No idea, absolutely, I had to use the way that 100% works - direct arguments to the CMake itself, but some I found:
https://stackoverflow.com/questions/16044020/gcc-and-linking-environment-variables-and-flags

Reply:
CMake is also can `CMAKE_C_COMPIER` and `CMAKE_CXX_COMPILER`, also `CMAKE_MAKE_PROGRAM`